### DISCUSSION TITLE
Using jsPsych and p5.js Libraries to Program an Experiment

### DISCUSSION THREAD

**Discussion by ghost** [jspsych/jsPsych#2056] [Q&A] [answered]
Hello,

I haven't dug too deep into this yet, but I was wondering if anyone has successfully used both jsPsych and p5.js libraries to create an experiment? 

The experiment I am programming requires displaying images and animating a spinning wheel based on the user's input. I am thinking about using p5.js for displaying the images and animation and jsPsych for the user interactions and timeline. 

Alternatively, does anyone happen to know how to animate/move images in jsPsych?

**Comment by GEJ1** [comment, marked as answer]
Hi @rmtham ,

Maybe you can check @kurokida 's [psychophysics plugin](https://jspsychophysics.hes.kyushu-u.ac.jp/)

Example: https://www.hes.kyushu-u.ac.jp/~kurokid/jspsychophysics/demos/mouse_event.html

**Reply by kurokida** [reply]
Thank you for introducing my plugin!

@rmtham , please see https://jspsychophysics.hes.kyushu-u.ac.jp/

I think my plugin is useful to you although p5.js is not used.

**Comment by Andre3582** [comment]
Hey,

Not sure if it is relevant to you anymore, but I have just created a jsPsych plugin that allows you to copy paste any p5.js sketch into a jsPsych experiment as a trial. I came across your question when looking for the same thing, but as I couldn't find it I did it myself ;)

see the repo here: https://github.com/Andre3582/jspsych-p5js-plugin
